src/scripts/tests/__snapshots__/import-open-api.test.ts.snap

```diff
@@ -12,6 +12,12 @@ export type Pet = NewPet & {id: number};
 
 export interface NewPet {name: string; tag?: string}
 
+export type CatOrDog = Cat | Dog;
+
+export interface Cat {type: \\"cat\\"; breed: \\"labrador\\" | \\"carlin\\" | \\"beagle\\"}
+
+export interface Dog {type: \\"dog\\"; breed: \\"saimois\\" | \\"bengal\\" | \\"british shorthair\\"}
```

Comment:
Why the double escape?

Reply:
It's a snapshot ^^ I'm not responsible of jest formatting 😁

Reply:
I don't really care about the formatting. I'm just wondering if/how it actually works:

![image](https://user-images.githubusercontent.com/9947422/68867226-c4feab80-06f5-11ea-83dc-f15d7e37f084.png)

Reply:
![image](https://user-images.githubusercontent.com/1761469/68875513-414bbb80-0703-11ea-9c45-f77666318c02.png)

It's not a typescript file! It's a SNAPSHOT 😅